### DISCUSSION TITLE
Connection to pg over ssh

### DISCUSSION THREAD

**Discussion by NiknamankiN** [jackc/pgx#1675] [Q&A] [answered]
Hello, I'm a newbie for go, could you provide any examples of establishing a connection to postgresql over ssh using pgxpool? Unfortunately, I could not find a working example myself...

**Comment by jackc** [comment, marked as answer]
https://github.com/jackc/tern has ssh tunneling built in. https://github.com/jackc/tern/blob/3b4b9b73743467e62d1214c28b8ce7d952ca14de/main.go#L141-L150 is a good starting place in the code.

**Reply by NiknamankiN** [reply]
Thank you!
